app/views/thesis/confirm.html.erb

```diff
@@ -0,0 +1,13 @@
+<h3>Success!</h3>
+<h4>A copy of your thesis information has been emailed to you.</h4>
```

Comment:
`will be` seems better here as technically it's an async queue and if it gets busy may take a bit plus email is slow and...